packages/mobile/src/screens/chats-screen/ChatsScreen.tsx

```diff
@@ -24,18 +24,20 @@ export const ChatsScreen = () => {
 
   return (
     <Screen>
-      <View>
-        <Text>You have {chats?.length} chats:</Text>
-        {messages.map((message) => {
-          return (
-            <View key={message.message_id}>
-              <Text>{message.sender_user_id} says:</Text>
-              <Text>{message.message}</Text>
-              <Text>{message.created_at}</Text>
-            </View>
-          )
-        })}
-      </View>
+      <ScreenContent>
```

Comment:
yeah could we just make the offline thing an option for screen? and default it to "not offline capable", and just the few screens that need it, we go `<Screen allowOffline />` or something?

Reply:
Yes, I think that's worth doing. I'll circle back to this tonight and do it that way.